test/e2e/network/loadbalancer.go

```diff
@@ -422,8 +422,10 @@ var _ = SIGDescribe("LoadBalancers", func() {
 			s.Spec.Ports[0].Port++
 		})
 		framework.ExpectNoError(err)
-		if int(udpService.Spec.Ports[0].Port) == svcPort {
-			framework.Failf("UDP Spec.Ports[0].Port (%d) did not change", udpService.Spec.Ports[0].Port)
+		svcPortOld := svcPort
+		svcPort = int(udpService.Spec.Ports[0].Port)
+		if svcPort == svcPortOld {
```

Comment:
I carried over the old bug, it has to fail if the ports are the same

Reply:
:)